.github/workflows/graphql.yml

```diff
@@ -0,0 +1,85 @@
+name: GraphQL
+on:
+  workflow_call:
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  graphql:
```

Comment:
The list of weblog variants is different and that felt pretty awkward to roll into the "main" jobs... It'd have required a lot of `indludes` or `excludes` in the matrix...

Reply:
You can configure the ignore variants and tests in the manifest files and add your scenario to the main job using an "if" like we are doing for other scenarios

Reply:
I had discussed this with @cbeauchesne... Using the manifest to configure the ignore list here effectively means copying the same set of ignore directives in a large amount of places, because these is effectively a deny list; and in this case we'd rather need an allow-list (those variants are ONLY relevant for GraphQL tests; no other).

I feel like using manifest exclusions would effectively be strictly worse in terms of developer experience...

Reply:
Is it possible to have several GraphQL deps in one weblog? 

One convenient way to tackle this use case is to expose a graphQL endpoi=nt, with a parameter to choose the engine. The test class will look likes : 

```python
def setup_test_graphql():
   if library == "nodejs":
      engines = ["apollo-server-express"] # and even by variant if needed
   elif library = "golang":
       engines = ["gqlgen", "graph-gophers", "graphql-go"]
    else:
        raise Exception("not implemented")
   
   for engine in engines:
       requests.append(get(f"/graphQl?engine={engine}") 
```

The issue with new weblog is mainly the CI time, and the fact we need to update lot of other CI.

If we find a way to integrate this test in the default (or existing) scenario, it a big win.

Reply:
I get the reason why you'd rather not use that strategy with the extra workflow; but if I want to go the way you suggest I have to either:
- Present *ALL 3* GraphQL engines behind *ALL 4* existing Go weblogs
- Exclude 3 of the Go weblogs on ALL GraphQL test scenarios (existing & future)

Neither of these is particularly sexy to me, and we have already discussed this over zoom and you seemed fine with the approach I've taken here. But fine, I yield and will flip this implementation over in hopes this can be merged sooner rather than later.

Reply:
It also strikes me that the approach of "hiding GraphQL behind the current weblogs" effectively makes it impossible to visualize contrib. coverage & parity properly from the reports...

Some GraphQL features may have varying degrees of support from contrib to contrib due to limitations imposed by the underlying engine's API (e.g, some of them would make it "realtively easy" to implement request blocking, while others might make it flat out impossible -- we'd only be able to report that they either all work, or some fails; but not have more detail than this ; which is possible with the separate weblog+scenario approach I had chosen here)

Reply:
Got offline (Slack) confirmation from @cbeauchesne that we do have valid reasons to separate. Could the PR be approved now?

Reply:
> we have already discussed this over zoom and you seemed fine with the approach I've taken here.

Ouch, please forgive me, too many discussions 😅 

I confirm what I said on slack : as long as you're aware with the pro & con of using another weblog, it's up to you to decide.